Don't attempt to download an agent binary when building the port.

--- src/ruby_supportlib/phusion_passenger/config/install_agent_command.rb.orig	Mon Jul 25 19:56:07 2016
+++ src/ruby_supportlib/phusion_passenger/config/install_agent_command.rb	Tue Nov  1 13:18:15 2016
@@ -50,9 +50,7 @@ module PhusionPassenger
         parse_options
         initialize_objects
         sanity_check
-        if !download
           compile
-        end
       end
 
     private
